Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -673,8 +673,14 @@ extension CarPlayManager {
         previewMapTemplate.showTripPreviews([modifiedTrip], textConfiguration: previewText)
         
         if currentActivity == .previewing {
-            interfaceController.safePopTemplate(animated: false)
-            interfaceController.pushTemplate(previewMapTemplate, animated: false)
+            if #available(iOS 14.0, *) {
+                interfaceController.popTemplate(animated: false) { _, _ in
+                    interfaceController.pushTemplate(previewMapTemplate, animated: false)
+                }
+            } else {
+                interfaceController.safePopTemplate(animated: false)
```

Comment:
It looks like `safePopTemplate(animated: Bool) {` contains [the check](https://github.com/mapbox/mapbox-navigation-ios/blob/lts/v2/Sources/MapboxNavigation/CPInterfaceController.swift) `== 1`. Should it be changed to `<= 1`? In order to avoid possible pop with 0 elements?